src/main/java/bio/terra/cda/app/controller/QueryApiController.java

```diff
@@ -89,4 +94,25 @@ private String createNextUrl(String jobId, int offset, int limit) {
 
     return sendQuery(querySql, dryRun);
   }
+
+  @Override
+  public ResponseEntity<QueryResponseData> uniqueValues(String version, @Valid String body) {
+
+    String bqTable = "gdc-bq-sample";
```

Comment:
Instead of hardcoding this, you should be able to get it from `applicationConfiguration.getBqTable()`. It looks like `version` is ignored, so you need something like what's done in `bulkData()`:
```
    String querySql = "SELECT * FROM " + applicationConfiguration.getBqTable() + "." + version;
```